src/app/modules/members/components/members-list/members-list-table/member-status/member-status.component.scss

```diff
@@ -0,0 +1,28 @@
+@import 'src/styles/_variables.scss';
+
+.base {
+  border: 2px solid;
+  border-radius: 4px;
+  padding: 3px 6px;
+}
+
+
+.new {
+  @extend .base;
+  color: $secondary;
+  border-color: $secondary;
+}
+
+.approved {
+  @extend .base;
+  color: $success;
+  border-color: $success;
+}
+
+.rejected {
+  @extend .base;
+  color: $danger;
+  border-color: $danger;
+}
```

Comment:
May be replaced with `border-2 border-XXX color-XXX` & padding classes _possibly_

Reply:
@Pav-el class="border-2" [ngClass]="{'border-success text-success': status === 'new' ......}" try something like that